platform/broadcom/saibcm-modules/systems/linux/kernel/modules/Makefile

```diff
@@ -24,7 +24,7 @@ LOCALDIR = systems/linux/kernel/modules
 
 include ${SDK}/make/Make.config
 
-subdirs=shared uk-proxy bcm-diag-full bcm-core bcm-net bcm-diag
+subdirs=
```

Comment:
why removed?

Reply:
the "systems" folder is copied as is from "sdk-gpl-6.5.19" source. 

There were some changes added back to the files copied. If this change has to be there then I can add it back.